docs/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
You can remove this file, because it will be accessible on marmelab.com/sedy/package.json